server/lib/transforms/removeInvalidRelated.js

```diff
@@ -9,10 +9,16 @@ module.exports = ($, warnings) => Promise.resolve()
 	.then(() => Promise.all($(relatedSelector).map((index, el) => {
 		const $el = $(el);
 		const href = $el.find(linkSelector).attr('href');
-		return fbApi.get({id: href, type: 'related'}).then(({og_object}) => {
-			if(og_object.type !== 'article') {
+		if(!href) {
```

Comment:
@quarterto I had to add these checks, as the `related-box` transform is adding some empty links. E.g. article `735eb4c8-998f-11e5-9228-87e603d47bdc` has:
```html
<ul class="op-related-articles" title="Further coverage">
    <li>
    </li>
</ul>
```

Can you look into this please?